### PR TITLE
Add job titles in pt_BR

### PR DESCRIPTION
**Pull request by brunodevel**

### CODE REVIEW

**Comment by ppelgrims** [comment]
The problem which caused the tests to fail is fixed. You can build again by closing and reopening this PR, rebasing and pushing, or by empty committing to this branch.

**Comment by brunodevel** [comment]
Thanks @ppelgrims . I'm reopening the PR now.

File: src/Faker/Provider/pt_BR/Company.php [hunk 1]
```diff
@@ -14,6 +14,25 @@ class Company extends \Faker\Provider\Company
         '{{lastName}} Comercial Ltda.'
     );
 
+    protected static $jobTitleFormat = array(
```
Comment (fzaninotto) [inline src/Faker/Provider/pt_BR/Company.php]: Can you please add a source in phpDoc?

Comment (brunodevel) [reply src/Faker/Provider/pt_BR/Company.php]: Done @fzaninotto.

File: src/Faker/Provider/pt_BR/Company.php [hunk 2]
```diff
@@ -14,6 +14,30 @@ class Company extends \Faker\Provider\Company
         '{{lastName}} Comercial Ltda.'
     );
 
+    /**
+     * @link https://www.mundovestibular.com.br/articles/15937/1/Os-50-melhores-empregos-e-os-50-piores/Paacutegina1.html
```
Comment (fzaninotto) [inline src/Faker/Provider/pt_BR/Company.php]: Unfortunately the content of this page is copyrighted, meaning you can't copy it and publish it under a MIT license. 

I'm sorry, but I can't accept this PR for legal reasons.

**Comment by marcelogarbin** [comment]
@fzaninotto Hello, how are you? I could check this pull https://github.com/fzaninotto/Faker/pull/1910, I made the list according to the government standard, I believe there is no problem using it, because it is used in general by everyone.

**Pull request closed without merging**